app/quizzes/[id]/choice-form.tsx

```diff
@@ -54,12 +59,11 @@ export default function ChoiceForm({ quizId }: { quizId: number }) {
             />
             <h2 className="grow font-bold">{idx + 1}번 선택지</h2>
           </label>
-          <MarkDown wrapLine={true} wrapLongLines={true}>
-            {choice.description}
-          </MarkDown>
+          <MarkDown wrapLongLines={true}>{choice.description}</MarkDown>
```

Comment:
p5: 이렇게 해도 괜찮을 것 같네요..!

```suggestion
          <MarkDown wrapLongLines >{choice.description}</MarkDown>
```